src/watts_ui/etc/watts.sch

```diff
@@ -0,0 +1,370 @@
+            }
+            extra_inputs{
+                Description = "[optional] List of extra (non-templated) input files that are needed"
+                InputTmpl="sonarray"
+                MinOccurs=0
+                MaxOccurs=1
+                value{
+                    MinOccurs=0
+                    MaxOccurs=NoLimit
+                }
+            }
+            extra_template_inputs{
+                Description = "[optional] List of extra templated input files that are needed"
+                InputTmpl="sonarray"
+                MinOccurs=0
+                MaxOccurs=1
+                value{
+                    MinOccurs=0
+                    MaxOccurs=NoLimit
+                }
+            }
+            scores{
+                Description = "[optional] List of scores for OpenMC tallies"
+                InputTmpl="sonarray"
+                MinOccurs=0
+                MaxOccurs=1
+                value{
+                    MinOccurs=0
+                    MaxOccurs=NoLimit
+                }
+            }
+            score_names{
+                Description = "[optional] List of representative names for scores"
+                InputTmpl="sonarray"
+                MinOccurs=0
+                MaxOccurs=1
+                value{
+                    MinOccurs=0
+                    MaxOccurs=NoLimit
+                }
+            }
+            output{
+                Description = "[optional] List of output files"
+                InputTmpl="sonarray"
+                MinOccurs=0
+                MaxOccurs=1
+                value{
+                    MinOccurs=0
+                    MaxOccurs=NoLimit
+                }
+            }
+            show_stderr{
+                MinOccurs=0
+                MaxOccurs=1
+                ValType=String
+            }
+            show_stdout{
+                MinOccurs=0
+                MaxOccurs=1
+                ValType=String
+            }
+            plotfl_to_csv{
+                MinOccurs=0
+                MaxOccurs=1
+                ValType=String
+            }
+            extra_args{
+                Description = "[optional] List of extra arguments"
+                InputTmpl="sonarray"
+                MinOccurs=0
+                MaxOccurs=1
+                value{
+                    MinOccurs=0
+                    MaxOccurs=NoLimit
+                }
+            }
+            conv_channel{
+                MinOccurs=0
+                MaxOccurs=1
+                ValType=String
+            }
+            conv_primar4{
+                MinOccurs=0
+                MaxOccurs=1
+                ValType=String
+            }
+            auto_link_files{
+                MinOccurs=0
+                MaxOccurs=1
+                ValType=String
+            }
+        }
+    }
+    workflow_dir{
+            Description = "[Required] Workflow directory: Dir where all files are located in. Necessary if have multiple input files."
```

Comment:
I don't understand why this option is required.

Reply:
Removed`workflow_dir`